lib/gui/models/drives.js

```diff
@@ -20,103 +20,94 @@
  * @module Etcher.Models.Drives
```

Comment:
I believe we can replace `redux-localstorage` with something like `redux-persist` maybe, but @jviotti correct me if I'm wrong.